### ISSUE TITLE
[FEATURE REQUEST] - TO HAVE MORE CHOICES OF APPS

### ISSUE THREAD

**Issue by bandinfinite** [Atlas-OS/Atlas#734]
### What is your Feature Request regarding to?

The Atlas OS

### Is your feature request related to a problem? Please describe.

No, just a suggestion

### Describe the solution you would like.

To have ShareX, qBitTorrent, Neat Download Manager (an alt to IDM, performs better), Hard Disk Sentinel (for error checking), KeePass, Revo Uninstaller (for adament programs), GIMP, LibreOffice, Sumatra PDF (very very very lightweight pdf viewer)

### Describe alternatives you have considered.

Sometimes, scoop too is nice than choco

### Requisites

- [X] This is not a support issue or a question. For any support, questions or help, join our [Discord server](https://discord.atlasos.net).
- [X] I performed a [cursory search of the issue tracker](https://github.com/Atlas-OS/Atlas/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I understand that not filling out this template will lead to the issue being closed.

**Comment by PencilNavigator** [comment]
Your recommendation is nice, however these will not be included in our install since it's basicly extra size for our playbook and not all users is going to need it.

Again, Scoop is removed due to it's instability and we don't need two package managers on atlas. If you want scoop install it yourself.

**Comment by bandinfinite** [comment]
Oh okay , I didn't knew about that

**Comment by JayXTQ** [comment]
Following on from what Pencil said, we use Chocolatey because it is easy and has lots of software inside of it, so no need for scoop built in. We do recommend scoop though which is why in the next update there will be a shortcut for it! Also, you can add those software to the install script yourself if you wish, just make a PR. And also @PencilNavigator it doesn't matter about GPLv3 compatibility here because we don't give the software on install, only when requested.

**Comment by PencilNavigator** [comment]
ok i misunderstood it, a URL shortcut wouldn't harm yeah?

**Comment by JayXTQ** [comment]
no, he is talking about adding those apps to `Install Software.ps1`
